### PR TITLE
Propose pull request template

### PR DESCRIPTION
**Pull request by rosibaj**
Proposal for a team PR template! Proposal is included.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+**Description of changes**
+
+Please add a brief description of the changes here.
+
+**Checklist before requesting review:**
+
+- [ ] Verified implemented page matches specified design sizes, fonts, & component styling. Copy should be spelled correctly.
```
Comment (joneubank) [inline .github/pull_request_template.md]: Bit wordy, perhaps something like:

[ ] Matches design:
    * component sizes, spacing, and styless
    * font size, weight, colour
    * spelling has been double checked

Comment (joneubank) [reply .github/pull_request_template.md]: ^ intended to be a nested list. comment formatting limited maybe

**Comment by joneubank** [comment]
Perhaps:
[ ] New components have React Storybook stories

**Comment by joneubank** [comment]
In looking at other PR templates, here are some things not included in this template... and I'm not sure we should, just ideas:

* Type of change (Bug, Styling, New Feature)
* Link to ticket and/or designs
* Steps to test

**Comment by rosibaj** [comment]
@joneubank good suggestions updated - im not sure about tests to steps.  thoughts @d8660091 @hlminh2000 @ciaranschutte would you guys find that helpful?

**Comment by joneubank** [comment]
Honestly I think it would be ignored in like 90% of PRs. Maybe leave from template.

**Pull request merged**